### DISCUSSION TITLE
Difference of Life cycles of Battery at 100% and 80% DOD

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#638] [Q&A] [answered]
Ich habe eine Batterie gefunden, die den Unterschied der Life Cycles angibt, abhaengig vom DOD% (Depth of Discharge = Tiefe der Entladung).

mehr als 6000 Zycklen klingt gut bei 80% DOD aber dann sah ich auch "bis 80% Ladung" (also die Batterie wird von 80% bis zu 20% benutzt, also 40% der Batterie bleibt unbenutzt 😢 )
Weiss vielleicht jemand, ob die Maximale Aufladung (bis zu 80%) eine grosse Role spielt? 
![Typical Life Cycles examples from TEZE Batteries](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/199912a7-f71a-45e8-85cb-2dca5520149a)

**Comment by spcqike** [comment]
Heißt 80% DOD nicht, dass sie zu 80% entladen wird? Also von 80 bis 0?


Zur Lebenszeit:


![](https://www.powertechsystems.eu/wp-content/uploads/sites/6/2019/09/EN-DoD-vs-Cycles-PowerBrick-PRO-version.png)

https://www.powertechsystems.eu/home/tech-corner/lithium-iron-phosphate-lifepo4/?amp

**Reply by Manos1966** [reply]
> Heißt 80% DOD nicht, dass sie zu 80% entladen wird? Also von 80 bis 0?

Jain... 
Sie wird sie zu 80% entladen wie du geschrieben hast, aber von 100%.
Also 80% DOD bedeutet die Batterie wird bis zu 20% entladen.

Danke fuer das Bild und vor allem fuer den Link! jetzt blicke ich (etwas besser) durch (und weiss, was 2C bedeutet):

The actual number of cycles that can be performed depends on several factors:
* Quality of Lithium Cell
* Level of power in C-Rate (1C rate means full discharge or charge in 1 hour, 2C is the same but in half an hour)
* Depth of Discharge (DOD)
* Operational environment : temperature, humidity, etc.

In standard environment, and for 1C cycles, we can get from the chart the below life cycle estimation for LFP :
3 000 cycles at 100% DoD
4 500 cycles at 80% DoD
10 000 cycles at 60% DoD
etc.
It should be noted that following the number of completed cycle, the batteries still have a nominal capacity > 80% of the original capacity.

**Reply by Sorbat** [reply, marked as answer]
Also ich habe mich das auch bei der pylontech schon gefragt.

Aber wenn man die Batterie rund 220 Tage/ Jahr nutzt und sie je Tag einen Ladezyklus durchläuft, dann hält sie 16 Jahre bei 4000 Zyklen und 27 Jahre bei 6000 Zyklen?

Will man nicht irgendwann auch gegen aktuelle technik tauschen oder so?

**Reply by Manos1966** [reply]
OK, so gesehen, ich kann wirklich meine Pylontechs voll ausnutzen, 16 Jahre klingt ausreichend.... 😄

**Comment by hoschiking** [comment]
Mit 80% ist in der Regel der SOC von 10-90% gemeint, also weder oben noch unten ans Limit gehen.
Beides ist für die Zellenlebensdauer nicht förderlich.
Das kann aber je nach Hersteller variieren und sollte in einem guten Datenblatt aufgeführt sein.
